.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,25 @@
-<!--
-Please read and follow these instructions before creating and submitting a pull request:
+<!-- Thanks for contributing! -->
 
-- If you're fixing a bug, ensure you add unit tests to prove that it works.
-- Before adding a feature, it is best to create an issue explaining it first. It would save you some effort in case we don't consider it should be included in node-fetch.
-- If you are reporting a bug, adding failing units tests can be a good idea.
--->
+## Purpose
+...
 
-**What is the purpose of this pull request?**
+## Changes
+...
 
-- [ ] Documentation update
-- [ ] Bug fix
-- [ ] New feature
-- [ ] Other, please explain:
+## Additional information
+...
 
-**What changes did you make? (provide an overview)**
 
-**Which issue (if any) does this pull request address?**
+___
 
-**Is there anything you'd like reviewers to know?**
+<!-- Mark what you have done, Remove unnecessary ones. Add new tasks that may fit (like TODO's) -->
+- [ ] I prefixed the PR-title with `docs: `, `fix(area): `, `feat(area): ` or `breaking(area): `
```

Comment:
At the moment, some of the currently open PRs follow this rule and the others don't. Since most of the commits currently on `master` don't use this convention, it would probably make sense to remove this rule.

Reply:
We could start normalizing it?

Reply:
I'm personally against conventional commit messages but I digress. It depends what the other maintainers also want.

Reply:
I do not have any strong opinion either. 
We can do whatever the next maintainer/reviewer has to say about it.

Reply:
Personally one thing I think is hard when contributing to many different projects is keeping track on what to put in the `area` part, and sometimes a change touches multiple parts of the library. I also have never understood what it's used for.

In my own projects I use [emoji-commit](https://github.com/LinusU/emoji-commit) which would be the equivalent of only using `docs: `, `fix: `, `feat: `, or `breaking: ` (although `docs` might not be the best since it doesn't cover e.g. linting, or other changes that doesn't affect the published package). I then have scripts which automatically bumps the appropriate patch/minor/major, and generates the change log.

If we are doing anything with the information, e.g. automatic bump/publish/changelog, then it's of course great. But if not, then the benefit is not as obvious. Especially the `(area)` part, would love some info on what the purpose of that is!

Anyhow, I don't have a strong opinion either way ☺️

Reply:
> In my own projects I use emoji-commit

lol, so that is why some commits messages looks so funky sometimes. 

> If we are doing anything with the information, e.g. automatic bump/publish/changelog, then it's of course great. But if not, then the benefit is not as obvious.

Agree with you here. It's mostly for automation. My intention was that it would be easy for us reviewer to easily know if a PR is a bugfix, new feature or if something is breakage. A simple github label would be sufficient.

I was thinking area could be something like:
- feat(Body.formData) - Support decompiling bodies back to FormData
- fix(FormData Encoder) - Made the encoder more spec compatible

Tried doing a little bit like Deno's PR: https://github.com/denoland/deno/pulls?q=is%3Apr+is%3Aclosed

Reply:
Can remove it... There might be times where the prefix also don't make sense.